docs/en/connector-v2/source/Http.md

```diff
@@ -124,6 +125,18 @@ connector will generate data as the following:
 
 the schema fields of upstream data
 
+### json_field [Config]
+
+The Wildcards for jsonpath like
```

Comment:
Add example json, and how to use `json_field` with example json, like parse data in list. And if we have `json_field`, do we need config schema or not? The answer of these question should be found in document.